libr/anal/cc.c

```diff
@@ -74,7 +74,7 @@ R_API void r_anal_cc_reset (RAnalCC *cc) {
 R_API char *r_anal_cc_to_string (RAnal *anal, RAnalCC* cc) {
 	RSyscallItem *si;
 	RAnalFunction *fcn;
-	char str[1024], buf[64];
+	char str[4096], buf[64];
```

Comment:
4096 is too big to fit in the stack.. i mean its ok for linux and mac, but can fail in some bsds and windows who have smaller stacksizes. better use the heap or use r_str_newf instead of fixed-sized buffers